05a-run_ica.py

```diff
@@ -14,50 +14,56 @@
 
 import config
 
-# Here we always process with the 1 Hz highpass data (instead of using
-# l_freq) because ICA needs a highpass.
 
-
-grads_rejection_limit = 4000e-13
-mags_rejection_limit = 4e-12
 decim = 11  # do not touch this value unless you know what you are doing
 
-
 def run_ica(subject, tsss=config.mf_st_duration):
     print("Processing subject: %s" % subject)
-    
+
     meg_subject_dir = op.join(config.meg_dir, subject)
-   
+
     epochs_fname = op.join(meg_subject_dir,
-                            config.base_epochs_fname.format(**locals()))
+                           config.base_epochs_fname.format(**locals()))
     epochs_for_ICA_fname = op.splitext(epochs_fname)[0] + '_for_ICA.fif'
```

Comment:
@massich I know that the file should end with epo, but I don't know how to do that.